cluster/examples/kubernetes/nfs/nfs.yaml

```diff
@@ -22,9 +22,11 @@ metadata:
   name: rook-nfs
   namespace: rook-nfs
 spec:
+  serviceAccountName: rook-nfs
   replicas: 1
   exports:
-  - name: nfs-share
+  - name: share1
+    createStorageClass: true
```

Comment:
here's the other `createStorageClass` usage that can hopefully be removed now

Reply:
done